### ISSUE TITLE
python3

### ISSUE THREAD

**Issue by JFsanchezherrero** [neherlab/pan-genome-analysis#27]
Dear developers,
I wonder if you are thinking in implementing a python3 version of this tool.

Thank you very much

**Comment by rneher** [comment]
yes. we are currently debating whether to do this as part of a bigger overhaul or a simply py23 conversion.

**Comment by JFsanchezherrero** [comment]
Wow! Great!

And do you have any scheduling for that? short-term, mid-term, long-term, undefined-term?
I would be interested in using it within my pipeline but I would really appreciate if it is in python3. 
I can even help you somehow on the conversion from py2->3. 

Thanks

**Comment by rneher** [comment]
I just pushed a py3 branch. This is basically running the entire codebase through 2to3 and fixing byte vs unicode string issues. It runs on the test data, but isn't otherwise tested. It would be great if you could give it a go and check whether it works.

**Comment by JFsanchezherrero** [comment]
Great!
I will give it a try next week and let you know! Thank you very much!
